### PR TITLE
MGMT-15890: Add recert optional openshift-e2e-test-qe main presubmit

### PR DESCRIPTION
**Pull request by mresvanis**
This PR adds the `openshift-e2e-test-qe` chain as an optional `main` branch presubmit test.

### CODE REVIEW

**Comment by mresvanis** [comment]
/pj-rehearse pull-ci-rh-ecosystem-edge-recert-main-e2e-aws-ovn-single-node-recert-openshift-e2e-test-qe

**Comment by mresvanis** [comment]
/pj-rehearse pull-ci-rh-ecosystem-edge-recert-main-e2e-aws-ovn-single-node-recert-openshift-e2e-test-qe

File: ci-operator/config/rh-ecosystem-edge/recert/rh-ecosystem-edge-recert-main.yaml [hunk 1]
```diff
@@ -3,6 +3,10 @@ base_images:
     name: ubi
     namespace: ocp
     tag: "9"
+  tests-private:
```
Comment (omertuc) [inline ci-operator/config/rh-ecosystem-edge/recert/rh-ecosystem-edge-recert-main.yaml:6]: Was this included in this PR on purpose?

Comment (mresvanis) [reply ci-operator/config/rh-ecosystem-edge/recert/rh-ecosystem-edge-recert-main.yaml:6]: Yes, this image is required by the [openshift-extended-tests](https://github.com/openshift/release/blob/bd1d761d67bd097f4cf48f802f7316abbde1b134/ci-operator/step-registry/openshift-extended/test/openshift-extended-test-ref.yaml#L3C1-L3C1), which is part of the the [openshift-e2e-test-qe-chain](https://github.com/openshift/release/blob/bd1d761d67bd097f4cf48f802f7316abbde1b134/ci-operator/step-registry/openshift/e2e/test/qe/openshift-e2e-test-qe-chain.yaml#L8).

**Comment by mresvanis** [comment]
/pj-rehearse pull-ci-rh-ecosystem-edge-recert-main-cargo-check

**Comment by mresvanis** [comment]
/pj-rehearse pull-ci-rh-ecosystem-edge-recert-main-cargo-check
/pj-rehearse pull-ci-rh-ecosystem-edge-recert-main-e2e-aws-ovn-single-node-recert-parallel

**Comment by mresvanis** [comment]
/pj-rehearse ci/rehearse/rh-ecosystem-edge/recert/main/e2e-aws-ovn-single-node-recert-openshift-e2e-test-qe

**Comment by mresvanis** [comment]
/test all
/pj-rehearse pull-ci-rh-ecosystem-edge-recert-main-e2e-aws-ovn-single-node-recert-openshift-e2e-test-qe

**Comment by mresvanis** [comment]
/pj-rehearse pull-ci-rh-ecosystem-edge-recert-main-e2e-aws-ovn-single-node-recert-openshift-e2e-test-qe

**Comment by mresvanis** [comment]
/pj-rehearse pull-ci-rh-ecosystem-edge-recert-main-e2e-aws-ovn-single-node-recert-openshift-e2e-test-qe

**Comment by mresvanis** [comment]
/pj-rehearse pull-ci-rh-ecosystem-edge-recert-main-e2e-aws-ovn-single-node-recert-openshift-e2e-test-qe

**Comment by mresvanis** [comment]
/pj-rehearse pull-ci-rh-ecosystem-edge-recert-main-e2e-aws-ovn-single-node-recert-openshift-e2e-test-qe

**Comment by mresvanis** [comment]
/pj-rehearse pull-ci-rh-ecosystem-edge-recert-main-e2e-aws-ovn-single-node-recert-openshift-e2e-test-qe

**Comment by mresvanis** [comment]
/pj-rehearse pull-ci-rh-ecosystem-edge-recert-main-e2e-aws-ovn-single-node-recert-openshift-e2e-test-qe

**Comment by mresvanis** [comment]
/pj-rehearse pull-ci-rh-ecosystem-edge-recert-main-e2e-aws-ovn-single-node-recert-openshift-e2e-test-qe

**Comment by mresvanis** [comment]
/pj-rehearse pull-ci-rh-ecosystem-edge-recert-main-e2e-aws-ovn-single-node-recert-openshift-e2e-test-qe

**Comment by mresvanis** [comment]
/pj-rehearse pull-ci-rh-ecosystem-edge-recert-main-e2e-aws-ovn-single-node-recert-openshift-e2e-test-qe

**Comment by mresvanis** [comment]
/pj-rehearse pull-ci-rh-ecosystem-edge-recert-main-e2e-aws-ovn-single-node-recert-openshift-e2e-test-qe

**Comment by mresvanis** [comment]
/pj-rehearse pull-ci-rh-ecosystem-edge-recert-main-e2e-aws-ovn-single-node-recert-openshift-e2e-test-qe

**Comment by mresvanis** [comment]
/pj-rehearse pull-ci-rh-ecosystem-edge-recert-main-e2e-aws-ovn-single-node-recert-openshift-e2e-test-qe

**Comment by mresvanis** [comment]
/pj-rehearse pull-ci-rh-ecosystem-edge-recert-main-e2e-aws-ovn-single-node-recert-openshift-e2e-test-qe

**Comment by mresvanis** [comment]
/pj-rehearse pull-ci-rh-ecosystem-edge-recert-main-e2e-aws-ovn-single-node-recert-openshift-e2e-test-qe

**Comment by mresvanis** [comment]
/test all
/pj-rehearse pull-ci-rh-ecosystem-edge-recert-main-e2e-aws-ovn-single-node-recert-openshift-e2e-test-qe

**Comment by mresvanis** [comment]
/pj-rehearse pull-ci-rh-ecosystem-edge-recert-main-e2e-aws-ovn-single-node-recert-openshift-e2e-test-qe

**Comment by mresvanis** [comment]
/pj-rehearse pull-ci-rh-ecosystem-edge-recert-main-e2e-aws-ovn-single-node-recert-openshift-e2e-test-qe

**Comment by mresvanis** [comment]
/pj-rehearse pull-ci-rh-ecosystem-edge-recert-main-e2e-aws-ovn-single-node-recert-openshift-e2e-test-qe

**Comment by mresvanis** [comment]
/pj-rehearse pull-ci-rh-ecosystem-edge-recert-main-e2e-aws-ovn-single-node-recert-openshift-e2e-test-qe

**Comment by mresvanis** [comment]
/pj-rehearse pull-ci-rh-ecosystem-edge-recert-main-e2e-aws-ovn-single-node-recert-openshift-e2e-test-qe

**Comment by mresvanis** [comment]
/pj-rehearse pull-ci-rh-ecosystem-edge-recert-main-e2e-aws-ovn-single-node-recert-openshift-e2e-test-qe

**Comment by mresvanis** [comment]
/pj-rehearse pull-ci-rh-ecosystem-edge-recert-main-e2e-aws-ovn-single-node-recert-openshift-e2e-test-qe

**Comment by mresvanis** [comment]
/pj-rehearse pull-ci-rh-ecosystem-edge-recert-main-e2e-aws-ovn-single-node-recert-openshift-e2e-test-qe

**Comment by mresvanis** [comment]
/pj-rehearse pull-ci-rh-ecosystem-edge-recert-main-e2e-aws-ovn-single-node-recert-openshift-e2e-test-qe

**Comment by mresvanis** [comment]
/pj-rehearse pull-ci-rh-ecosystem-edge-recert-main-e2e-aws-ovn-single-node-recert-openshift-e2e-test-qe

**Comment by mresvanis** [comment]
/pj-rehearse pull-ci-rh-ecosystem-edge-recert-main-e2e-aws-ovn-single-node-recert-openshift-e2e-test-qe

**Pull request merged**